Tests/Java2SwiftTests/Java2SwiftTests.swift

```diff
@@ -114,6 +114,36 @@ class Java2SwiftTests: XCTestCase {
       ]
     )
   }
+
+  func testNestedSubclasses() async throws {
```

Comment:
I grabbed a different set of test classes out of `java.lang`; love 'em both. Could we drop the unnecessary `async` here?

Reply:
This got updated in the rebase I think (I had this in a partial working state for a few weeks, but couldn't find the time). updated